src/v/config/configuration.cc

```diff
@@ -3916,6 +3916,41 @@ configuration::configuration()
       std::chrono::milliseconds{1min},
       {.min = std::chrono::milliseconds{10s},
        .max = serde::max_serializable_ms})
+  , datalake_scheduler_block_size_bytes(
+      *this,
+      "datalake_scheduler_block_size_bytes",
+      "Size, in bytes, of a memory block reserved for record translation, as "
+      "tracked by the datalake scheduler.",
```

Comment:
nit: s/a/each? As is, it reads like all translation will reserve a single block of 4MiB

Reply:
yeah fair point

Reply:
done